packages/editor/src/plugins/root-with-optionals/renderer.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactNode } from 'react'
+
+export function RootWithOptionalsRenderer({ content }: { content: ReactNode }) {
+  return <>{content}</>
+}
```

Comment:
def. a bit verbose, but I don't mind for consistency / possible future features.